src/logic/calculate.js

```diff
@@ -0,0 +1,88 @@
+import operate from './operate';
+
+const calculate = (data, btnName) => {
+  let {
+    total, next, operation, content, priorNext, priorOp,
+  } = data;
+
```

Comment:
From the project requirement on number 7: the **data prop** should have three properties: total, next, and operation. 
Here's the link: https://microverse.pathwright.com/library/react-redux-v11/184438/path/step/106376018/